examples/kitchen-sink.yaml

```
@@ -434,3 +434,107 @@ resource:
+      service_mapping:
+        - peer: 1.2.3.4
+          service: FooService
+        - peer: 2.3.4.5
+          service: BarService
+    # Configure instrumentations following the http semantic conventions.
+    # 
+    # See http semantic conventions: https://opentelemetry.io/docs/specs/semconv/http/
+    http:
+      # Configure instrumentations following the http client semantic conventions.
+      client:
+        # Configure headers to capture for outbound http requests.
+        request_captured_headers:
+          - Content-Type
+          - Accept
+        # Configure headers to capture for outbound http responses.
+        response_captured_headers:
+          - Content-Type
+          - Content-Encoding
+      # Configure instrumentations following the http server semantic conventions.
+      server:
+        # Configure headers to capture for inbound http requests.
+        request_captured_headers:
+          - Content-Type
+          - Accept
+        # Configure headers to capture for outbound http responses.
+        response_captured_headers:
+          - Content-Type
+          - Content-Encoding
+  # Configure language-specific instrumentation libraries.
+  #
+  # Keys may refer to instrumentation libraries or collections of related configuration. Because there is no central schema defining the keys or their contents, instrumentation must carefully document their schema and avoid key collisions with other instrumentations.
+  #
+  # Configure C++ language-specific instrumentation libraries.
+  cpp:
+    # Configure the instrumentation corresponding to key "another_example_key".
+    example:
+      property: "value"
+  # Configure .NET language-specific instrumentation libraries.
+  dotnet:
+    # Configure the instrumentation corresponding to key "another_example_key".
+    example:
+      property: "value"
+  # Configure Erlang language-specific instrumentation libraries.
+  erlang:
+    # Configure the instrumentation corresponding to key "another_example_key".
+    example:
+      property: "value"
+  # Configure Go language-specific instrumentation libraries.
+  go:
+    # Configure the instrumentation corresponding to key "another_example_key".
+    example:
+      property: "value"
+  # Configure Java language-specific instrumentation libraries.
+  java:
+    # Configure the instrumentation corresponding to key "another_example_key".
+    example:
+      property: "value"
+  # Configure JavaScript language-specific instrumentation libraries.
+  js:
+    # Configure the instrumentation corresponding to key "another_example_key".
+    example:
+      property: "value"
+  # Configure PHP language-specific instrumentation libraries.
+  php:
+    # Configure the instrumentation corresponding to key "another_example_key".
+    example:
+      property: "value"
+  # Configure Python language-specific instrumentation libraries.
+  python:
+    # Configure the instrumentation corresponding to key "another_example_key".
```

```suggestion
    # Configure the instrumentation corresponding to key "example".
```

Repeated a couple of times.

---

Thanks